src/medVtkInria/vtkDataManagement/vtkMetaSurfaceMesh.cxx

```diff
@@ -516,8 +517,45 @@ void vtkMetaSurfaceMesh::ReadMeshFile (const char* filename)
   {
     outputmesh->GetPointData()->AddArray (pointarray);
   }
-  
+
+  vtkIdList* idlist = vtkIdList::New();
+  unsigned int ids[3];
+
+  // N.B. in vtkPolyData, the order of cells is capital:
+  // vertices then lines then triangles.
+ 
   file >> str;
+  // For now ignore vertices, find edges first
+  while( (strcmp (str, "Edges") != 0) && (file.good()) )
+  {
+    file >> str;
+  }
+
+  unsigned int NEdges = 0;
+  if(file.good())
+  {
+    // read edges
+    file >> NEdges;
+
+    outputmesh->Allocate (NEdges);
+    cellarray->Allocate(NEdges);
+
+    for (unsigned int i = 0; i < NEdges; ++i)
+    {
+      file >> ids[0] >> ids[1] >> ref;
+      idlist->Initialize();
+      idlist->InsertNextId (ids[0]-1);
+      idlist->InsertNextId (ids[1]-1);
+   
+      outputmesh->InsertNextCell(VTK_LINE, idlist);
+      cellarray->InsertNextValue(ref);
+    }
+  }
+
+  file.close();
```

Comment:
`file.seekg(0);` should set the "cursor" at the beginning of the file, if needed